src/parser/jobs/sch/modules/ScholarCooldownDowntime.tsx

```diff
@@ -0,0 +1,23 @@
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
```

Comment:
Prefer to rename the import and use a sane name for the class.

```suggestion
import {CooldownDowntime as CoreCooldownDowntime} from 'parser/core/modules/CooldownDowntime'
```